docs-v2/pages/foundations/index.tsx

```diff
@@ -30,7 +30,7 @@ const FoundationsPage: React.FC<Props> = () => {
           flexDirection="column"
           justifyContent="flex-start"
           padding={majorScale(5)}
-          maxWidth={1200}
+          maxWidth={1024}
```

Comment:
Not related to this PR at all, but let's move these shared UIs out to templated components. Happy to do this in a follow-up.